drivers/led/ht16k33.c

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright (c) 2019 Henrik Brix Andersen <henrik@brixandersen.dk>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief LED driver for the HT16K33 I2C LED driver with keyscan
```

Comment:
It's good to document the limitations (if any) of the driver here

Reply:
Right. Anything in particular you think I should mention here?